demo/test.html

```diff
@@ -1,204 +1,287 @@
-          <option key="motorbike">motorbike</option>
-        </select>
-        <input id="trip-destination-tag" class="form-control mr-sm-2 mb-1" type="text" placeholder="Enter Destination Geofence Tag" style="width: 400px;"/>
-        <input id="trip-destination-externalId" class="form-control mr-sm-2 mb-1" type="text" placeholder="Enter Destination Geofence External Id" style="width: 400px;"/>
-        <button id="trip-start-btn" class="btn btn-primary">Start Trip</button>
-        <button id="trip-complete-btn" class="btn btn-primary">Complete Trip</button>
-      </form>
+<body>
+  <div class="links pl-3 pt-3">
+    <div class="btn-group" role="group" aria-label="Basic example">
+      <a href='/' role="button" class="std-link btn btn-secondary">radar.js</a>
+      <a href='/?min' role="button" class="min-link btn btn-secondary">radar.min.js</a>
     </div>
+  </div>
+
+  <div class="pl-3 pt-3">
+    <form id="track-once" class="form-inline mb-0">
+      <input id="publishable-key" class="form-control mr-sm-2" type="text" name="publishable-key"
+        placeholder="Enter Publishable Key" style="width: 500px;" />
+      <button id="track-btn" class="btn btn-primary">Call Track Once</button>
+    </form>
+  </div>
+
+  <div class="pl-3 pt-3">
+    <a href="javascript:void(0)" id="show-trip-options">Show trip options</a>
+    <form id="trip-form" class="form mb-0 d-none">
+      <input id="trip-id" class="form-control mr-sm-2 mb-1" type="text" placeholder="Enter Trip Id"
+        style="width: 400px;" />
+      <select id="trip-mode" class="form-control mr-sm-2 mb-1" style="width: 400px;">
+        <option key="" disabled>Select Mode</option>
+        <option key="car">car</option>
+        <option key="foot">foot</option>
+        <option key="bike">bike</option>
+        <option key="truck">truck</option>
+        <option key="motorbike">motorbike</option>
+      </select>
+      <input id="trip-destination-tag" class="form-control mr-sm-2 mb-1" type="text"
+        placeholder="Enter Destination Geofence Tag" style="width: 400px;" />
+      <input id="trip-destination-externalId" class="form-control mr-sm-2 mb-1" type="text"
+        placeholder="Enter Destination Geofence External Id" style="width: 400px;" />
+      <button id="trip-start-btn" class="btn btn-primary">Start Trip</button>
+      <button id="trip-complete-btn" class="btn btn-primary">Complete Trip</button>
+    </form>
+  </div>
 
-    <div class="loader pl-3 d-none">
-      <!-- By Sam Herbert (@sherb), for everyone. More @ http://goo.gl/7AJzbL -->
-      <svg width="44" height="44" viewBox="0 0 44 44" xmlns="http://www.w3.org/2000/svg" stroke="#007bff">
-        <g fill="none" fill-rule="evenodd" stroke-width="2">
+  <div class="pl-3 pt-3">
+    <a href="javascript:void(0)" id="show-matrix-options">Show Matrix API options</a>
+    <form id="matrix-form" class="form mb-0 d-none">
+      <input id="matrix-origins" class="form-control mr-sm-2 mb-1" type="text"
```

Comment:
@corypisano do you think this may be suited better on our API demo page? that's powered by our Web SDK today, so once we have this method we'd be able to layer a UI on top

Reply:
im good having it here as well, for internal use vs developer example